Munsa1/blog-app
# Validations, Model Specs & N+1 Problems

## Validations, Model Specs & N+1 Problems:

In the following pull request I add the following validations:

For the User model:
- Name must not be blank.
- PostsCounter must be an integer greater than or equal to zero.

For the Post model:
- Title must not be blank.
- Title must not exceed 250 characters.
- CommentsCounter must be an integer greater than or equal to zero.
- LikesCounter must be an integer greater than or equal to zero.

Further I make sure I:
- Add unit specs for all of your models' methods and validations.
- Add flash messages in the create actions in all your controllers.
- Make sure that the N+1 problem is solved when fetching all posts and their comments for a user.
